plugins/kubernetes-backend/package.json

```diff
@@ -40,6 +40,7 @@
     "@backstage/catalog-client": "^1.0.4-next.1",
     "@backstage/catalog-model": "^1.1.0-next.2",
     "@backstage/config": "^1.0.1",
+    "@backstage/core-plugin-api": "^1.0.3-next.0",
```

Comment:
Oh this should not be here - this would be a dependency from a backend plugin to a frontend core.